pkgs/applications/audio/lingot/default.nix

```diff
@@ -46,6 +46,6 @@ stdenv.mkDerivation rec {
     homepage = "https://www.nongnu.org/lingot/";
     license = lib.licenses.gpl2Plus;
     platforms = with lib.platforms; linux;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```

Comment:
```suggestion
    maintainers = [ ];
```

Reply:
Mhm. I was hoping, I could just commit these straight to the branch, it it seems like github no longer allows me to add those.

Reply:
This is a bad problem from GitHub: it does not allow edits from upstream maintainers when the PR comes from an organization.